modules/puzzles/module.py

```diff
@@ -52,3 +53,13 @@ def buildings_with(self, symbols):
         results = util.buildings_containing_symbols(symbols)
         self.respond('Er zijn %i gebouwen die de symbolen \'%s\' bevatten' % (len(results), symbols))
         self.respond(', '.join([str(x) for x in results]))
+
+    @command
+    def coordinates(self, numbers):
+        """
+        Geeft alle mogelijke locaties op de campus met coordinaten bestaande uit de gegeven getallen. Getallen dienen te worden gescheiden met komma's
+        """
+        results = util.brute_force_coordinates(numbers.split(','))
+        self.respond('Er zijn %i locaties op de campus met deze getallen.' % (len(results)))
+        if(results):
+            self.respond('\n'.join([('' + str(x[0]) + ', ' + str(x[1])) for x in results]))
```

Comment:
`('' + str(x[0]) + ', ' + str(x[1]))` kan denk ik mooier als `('%s, %s' % (x[0], x[1]))` of zelfs als`('%s, %s' % tuple(x))`

Reply:
Fixed